tests/parser/test_parser.py

```diff
@@ -76,7 +76,7 @@ def test_parse_md_details(caplog):
 
     # Check that doc has sentences
     assert len(doc.sentences) == 45
-    sent = doc.sentences[25]
+    sent = sorted(doc.sentences, key=lambda x: x.position)[25]
```

Comment:
Noticed some inconsistency in sorting this time around, so just sorting the sentences based on their position before testing so that it is deterministic.